tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_glusterfs_profile_info.py

```diff
@@ -236,8 +237,24 @@ def get_volume_profile_info(self, volName, cluster_id):
             )
             return ret_val
 
-    def process_volume_profile_info(self, volume):
+    def process_volume_profile_info(self, volume, etcd_client):
         volName = volume['name']
+        volId = volume['id']
+        volData = etcd_client.read(
```

Comment:
`self.etcd_client` should be used, no need for the argument

Reply:
Ack. Will change.